writing-guide.md

```diff
@@ -0,0 +1,86 @@
+# Vue Docs Writing Guide
+
+Writing documentation is an exercise in empathy. We're not describing an objective reality - the source code already does that. Our job is to help shape the relationship between users and the Vue ecosystem. This ever-evolving guide provides some rules and recommendations on how to do that consistently within the Vue ecosystem.
+
+## Principles
+
+- **A feature doesn't exist until it's well documented.**
+- **Respect users' cognitive capacity (i.e. brain power).** When a user starts reading, they begin with a certain amount of limited brain power and when they run out, they stop learning.
+  - Cognitive capacity is **depleted faster** by complex sentences, having to learn more than one concept at a time, and abstract examples that don't directly relate to a user's work.
+  - Cognitive capacity is **depleted more slowly** when we help them feel consistently smart, powerful, and curious. Breaking things down into digestible pieces and minding the flow of the document can help keep them in this state.
+- **Always try to see from the user's perspective.** When we understand something thoroughly, it becomes obvious to us. This is called _the curse of knowledge_. In order to write good documentation, try to remember what you first needed to know when learning this concept. What jargon did you need to learn? What did you misunderstand? What took a long time to really grasp? Good documentation meets users where they are. It can be helpful to practice explaining the concept to people in person before
```

Comment:
Added a slight rewording to avoid repeating the same adjectives

```suggestion
- **Always try to see from the user's perspective.** When we understand something thoroughly, it becomes evident to us. This is called _the curse of knowledge_. In order to write good documentation, try to remember what you first needed to know when learning this concept. What jargon did you need to learn? What did you misunderstand? What took a long time to really grasp? Proper documentation meets users where they are. It can be helpful to practice explaining the concept to people in person before
```

Reply:
I think "obvious" is actually closer to what I mean here than "evident". To me, "obvious" suggests not only understanding, but _ease_ of understanding. Once we learn something, we forget how difficult it was to learn and how difficult it may still be for others. "Evident" doesn't carry the same connotation.

Regarding "good" vs "proper", I think this is one of the occasions where repeating an adjective is actually OK (and even desirable). We're repeating an adjective in the same context, to reference a previous concept: "good documentation". Changing the second adjective to "proper" would suggest that "proper documentation" is a new concept separate from "good documentation". In general though, you're right! Repeating the same adjectives across different contexts (e.g. "during my interesting walk I saw an interesting cat") can make the writing less, um, interesting. 😜 However, this is also somewhat in conflict with our goal to keep language simpler. If our option is to replace a word with a much less common one, I think I'd prefer repeated adjectives over forcing users to expand their English vocabulary.